arangod/Agency/AgencyComm.h

```diff
@@ -372,6 +372,15 @@ struct AgencyWriteTransaction : public AgencyTransaction {
     preconditions.push_back(precondition);
   }
 
+  AgencyWriteTransaction(AgencyOperation const& operation,
+                         std::vector<AgencyPrecondition> const& precs) :
+    clientId(to_string(boost::uuids::random_generator()())) {
```

Comment:
Can't we use a standard tick value here?

Reply:
You mean using `ClusterInfo::uniqid()`? That itself uses an `AgencyWriteTransaction` to fetch unique ids from the agency, so this seems problematic to bootstrap.